javascript/examples/embed-pdf-template-builder.js

```diff
@@ -61,15 +60,13 @@ async function graphQLResponse (graphQLQuery) {
 async function createCast (variables) {
   const query = `
     mutation CreateCast(
-      $organizationEid: String,
       $title: String,
       $file: Upload!,
       $isTemplate: Boolean,
       $detectFields: Boolean,
       $allowedAliasIds: [String],
     ) {
       createCast (
-        organizationEid: $organizationEid,
```

Comment:
We don't need an org eid because it's found from the API key, is that right?

Reply:
yep exactly